scipy/stats/__init__.py

```diff
@@ -291,16 +312,36 @@
    normaltest
 
 Objects returned by some statistical tests
-==========================================
+------------------------------------------
 
 .. autosummary::
    :toctree: generated/
 
    BinomTestResult
 
 
+Masked statistics functions
+===========================
+
+.. toctree::
+
+   stats.mstats
+
+
+Quasi-Monte Carlo
```

Comment:
I was going to suggest in the QMC PR that putting this above masked statistics functions might improve its visibility.

Reply:
Agreed, good idea.